src/utils/aeternity.js

```diff
@@ -94,13 +100,21 @@ export const scanForWallets = async () => {
   });
 };
 
-const createOrChangeAllowance = async (tokenAddress, amount) => {
+export const tokenBalance = async (token, address) => {
```

Comment:
Currently, it is defined [here](https://github.com/aeternity/superhero-ui/blob/fc1d5d4eef9ba77b91b9e902ff16096da73ade88/src/store/index.js#L51-L56)